tests/ext/integrations/curl/distributed_tracing_curl_copy_handle.phpt

```diff
@@ -3,6 +3,7 @@ Distributed tracing headers propagate after curl_copy_handle()
 --SKIPIF--
 <?php if (!extension_loaded('curl')) die('skip: curl extension required'); ?>
 <?php if (!getenv('HTTPBIN_HOSTNAME')) die('skip: HTTPBIN_HOSTNAME env var required'); ?>
+<?php die('skip: disabled functionality: x-datadog-tags'); ?>
```

Comment:
I suggest to leave the test active (as it tests many things) and just removing the tags part

Reply:
done